features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/InternalCoreWriterProvider.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.trace
+
+/**
+ * Internal interface to provide the core writer to the tracer.
```

Comment:
Should we explain what is the meaning of the `core` here? The type of the returned writer is `com.datadog.trace.common.writer.Writer`, so it has no mention of the core.

Reply:
yup I will add an explanation there